src/popup/router/components/Button.vue

```diff
@@ -67,6 +65,34 @@ export default {
 @use '../../../styles/variables';
 @use '../../../styles/typography';
 
+.new-ui {
```

Comment:
I strongly recommend to not do this change. Components should be an encapsulated things and it's states should be only changed with the props. Only exception for this are the css variables.